polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -523,10 +532,48 @@ impl pallet_treasury::Config for Runtime {
 	type SpendPeriod = SpendPeriod; //Cannot be 0: Error: Thread 'tokio-runtime-worker' panicked at 'attempt to calculate the remainder with a divisor of zero
 	type Burn = (); //No burn
 	type BurnDestination = (); //No burn
-	type SpendFunds = (); //No spend, no bounty
+	type SpendFunds = Bounties;
+	type SpendOrigin = EnsureWithSuccess<EnsureRoot<AccountId>, AccountId, MaxBalance>;
 	type MaxApprovals = MaxApprovals; //0:cannot approve any proposal
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
-	type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>; // Same as kusama
+}
+
+parameter_types! {
+	pub const BountyDepositBase: Balance = 1 * TEER;
+	pub const BountyDepositPayoutDelay: BlockNumber = prod_or_fast!(4 * DAYS, 4 * MINUTES);
+	pub const BountyUpdatePeriod: BlockNumber = prod_or_fast!(90 * DAYS, 15 * MINUTES);
+	pub const MaximumReasonLength: u32 = 16384;
```

Comment:
Is 16kb enough for bounty proposals? Is this in our case just intended to be the encoded text for the proposal or something else? I suppose some upper bound needs to be chosen and if it is just characters then 16kb probably is sufficient.

Reply:
actually, I think it should be enough to carry a url of a github issue. but not more than that and certainly not an entire description, that would just be chain bloat, so I guess the current length is plenty. 
As people pay fees per length, I think it should be safe as is.

Reply:
Ok a link seems reasonable yea I agree anything more is probably worthy of a hash instead of just a bunch of description bytes. 

I did not think it was dangerous by any means just was more of a curiosity.